iam/terraform/account-specific/main/iam_groups.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_iam_group" "developers" {
+  name = "developers"
```

Comment:
Maybe this could be something like `log_viewers`? Just thinking of something that would say more about what the group does.